test/config/integration/certs/certs.sh

```diff
@@ -2,32 +2,32 @@
 
 set -e
 
-# Uncomment the following lines if you want to regenerate the private keys.
-# openssl genrsa -out cakey.pem 1024
-# openssl genrsa -out serverkey.pem 1024
-# openssl genrsa -out clientkey.pem 1024
-# openssl genrsa -out upstreamcakey.pem 1024
-# openssl genrsa -out upstreamkey.pem 1024
+generate_ca() {
+  openssl genrsa -out cakey.pem 1024
+  openssl req -new -key $1cakey.pem -out $1cacert.csr -config $1cacert.cfg -batch -sha256
+  openssl x509 -req -days 730 -in $1cacert.csr -signkey $1cakey.pem -out $1cacert.pem \
+    -extensions v3_ca -extfile $1cacert.cfg
+}
+
+generate_cert_key_pair() {
+  openssl genrsa -out $1key.pem 1024
+  openssl req -new -key $1key.pem -out $1cert.csr -config $1cert.cfg -batch -sha256
+  openssl x509 -req -days 730 -in $1cert.csr -sha256 -CA $2cacert.pem -CAkey \
```

Comment:
Same here, please always pass both `$1` and `$2`, i.e. `generate_cert_key_pair ca server`, `generate_cert_key_pair ca client`, `generate_cert_key_pair upstream upstream`.